packages/components/tests/dummy/app/templates/overrides/power-select.hbs

```diff
@@ -0,0 +1,284 @@
+{{page-title "PowerSelect Component"}}
+
+<h2 class="dummy-h2">PowerSelect</h2>
+
+<section>
+  <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
+
+  <p class="dummy-paragraph">
+    <a target="_blank" rel="noopener noreferrer" href="https://ember-power-select.com/">PowerSelect</a>
+    is a popular Ember add-on aiming to overcome some limitations of the
+    <code class="dummy-code">&lt;select&gt;</code>
+    tag. HashiCorp Design System only provides style overrides for this add-on to keep it in line with other form
+    components.
+  </p>
+
+  <p class="dummy-banner dummy-banner--info dummy-paragraph"><FlightIcon @name="info" />Note: These style overrides
+    assume the PowerSelect component is set up in your project using the default theme.</p>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="how-to-use"><a href="#how-to-use" class="dummy-link-section">§</a> How to use</h3>
+  <p class="dummy-paragraph">
+    To use this component in your application follow
+    <a target="_blank" rel="noopener noreferrer" href="https://ember-power-select.com">the getting started guide on the
+      add-on website</a>
+    then add the PowerSelect overrides as shown below.
+  </p>
+
+  <h4 class="dummy-h4">Overrides import</h4>
+  <p class="dummy-paragraph">If you are already using
+    <a
+      target="_blank"
+      rel="noopener noreferrer"
+      href="https://github.com/hashicorp/design-system/blob/main/packages/components/README.md"
+    >design-system-components</a>
+    in your project the overrides import would look something like this:</p>
+  {{! prettier-ignore-start }}
+  {{! template-lint-disable no-unbalanced-curlies }}
+  <CodeBlock
+    @code='
+      @import "@hashicorp/design-system-components";
+      @import "@hashicorp/power-select-overrides";
```

Comment:
_maybe_ I would be over-specific and explicit, and call the file `@hashicorp/design-system-power-select-overrides`

Reply:
agree, better over-specific than ambiguous – will update the file name (and references to it)